app-office/libreoffice/libreoffice-5.1.4.2.ebuild

```diff
@@ -265,9 +265,7 @@ pkg_pretend() {
 	if [[ ${MERGE_TYPE} != binary ]]; then
 		check-reqs_pkg_pretend
 
-		if [[ $(tc-getCC) == clang ]] ; then
-			: # ignore clang, which works
-		elif [[ $(gcc-major-version) -lt 4 ]] || {
+		if [[ $(gcc-major-version) -lt 4 ]] || {
```

Comment:
That doesn't look right, you basically reverted 2ed0f7a6012687058661e1cb93b51bb599719b98 and got rid of the check that was supposed to fix build with clang in the first place (even if it didn't work).